src/python/bin/aqueduct

```diff
@@ -501,27 +532,34 @@ def install_mysql():
                 ]
             )
         elif distro.id() == "centos" or distro.id() == "rhel":
-            execute_command(["sudo", "yum", "install", "-y", "python3-devel", "mysql-devel"])
+            execute_command(["sudo", "yum", "install", "-y",
+                            "python3-devel", "mysql-devel"])
         else:
             print("Unsupported distribution:", distro.id())
     elif system == "Darwin":
         cmd = ["brew", "install", "mysql"]
-        architecture = subprocess.Popen(["which", "-a", "brew"], stdout=subprocess.PIPE ).communicate()[0]
-        if architecture.startswith(b"/opt/homebrew"): # Using arm verison of brew
+        architecture = subprocess.Popen(
```

Comment:
Shouldn't we add the Mac mysql installation here instead?

Reply:
I didn't actually touch that code. Looks like the linter just reformatted things.

What I added here was the pip_cmd and the execute_cmd below (lines 547 and 548)

Reply:
yeah I mean move what you added in `src/python/aqueduct_executor/operators/connectors/data/execute.py` here?  I seems better to have all the installation be done at `aqueduct install` time rather than in `setup_connector()`?

Reply:
Actually come to think of it, we're going to need to run this for Linux as well. I've now moved the installation step for the python package to the bottom. Now each platform installs their prerequisites, then installs the pymysql package.

As far as the import stuff goes, we need to run this when we import the MySQLdb module, so that should probably stay in the execute.py step. I've included some comments there to explain what's going on as well.